cmd/subctl/show.go

```diff
@@ -0,0 +1,100 @@
+*/
+
+package subctl
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/submariner-operator/cmd/subctl/execute"
+	"github.com/submariner-io/submariner-operator/internal/show"
+)
+
+var (
+	// showCmd represents the show command.
+	showCmd = &cobra.Command{
+		Use:   "show",
+		Short: "Show information about Submariner",
+		Long:  `This command shows information about some aspect of the Submariner deployment in a cluster.`,
+	}
+	connCmd = &cobra.Command{
+		Use:     "connections",
+		Short:   "Show cluster connectivity information",
+		Long:    `This command shows information about Submariner endpoint connections with other clusters.`,
+		PreRunE: restConfigProducer.CheckVersionMismatch,
+		Run: func(command *cobra.Command, args []string) {
+			execute.OnMultiCluster(restConfigProducer, execute.IfSubmarinerInstalled(show.Connections))
+		},
+	}
+	epCmd = &cobra.Command{
+		Use:     "endpoints",
+		Short:   "Show Submariner endpoint information",
+		Long:    `This command shows information about Submariner endpoints in a cluster.`,
+		PreRunE: restConfigProducer.CheckVersionMismatch,
+		Run: func(command *cobra.Command, args []string) {
+			execute.OnMultiCluster(restConfigProducer, execute.IfSubmarinerInstalled(show.Endpoints))
+		},
+	}
+	gwCmd = &cobra.Command{
+		Use:     "gateways",
+		Short:   "Show Submariner gateway summary information",
+		Long:    `This command shows summary information about the Submariner gateways in a cluster.`,
+		PreRunE: restConfigProducer.CheckVersionMismatch,
+		Run: func(command *cobra.Command, args []string) {
+			execute.OnMultiCluster(restConfigProducer, execute.IfSubmarinerInstalled(show.Gateways))
+		},
+	}
+	ntwrkCmd = &cobra.Command{
+		Use:     "networks",
+		Short:   "Get information on your cluster related to Submariner",
+		Long:    `This command shows the status of Submariner in your cluster, and the relevant network details from your cluster.`,
+		PreRunE: restConfigProducer.CheckVersionMismatch,
+		Run: func(command *cobra.Command, args []string) {
+			execute.OnMultiCluster(restConfigProducer, show.Network)
+		},
+	}
+	versionCmd = &cobra.Command{
+		Use:     "versions",
+		Short:   "Shows Submariner component versions",
+		Long:    `This command shows the versions of the Submariner components in the cluster.`,
+		PreRunE: restConfigProducer.CheckVersionMismatch,
+		Run: func(command *cobra.Command, args []string) {
+			execute.OnMultiCluster(restConfigProducer, execute.IfSubmarinerInstalled(show.Versions))
+		},
+	}
+	allCmd = &cobra.Command{
+		Use:   "all",
+		Short: "Show information related to a Submariner cluster",
+		Long: `This command shows information related to a Submariner cluster:
+		      networks, endpoints, gateways, connections, broker and component versions.`,
+		PreRunE: restConfigProducer.CheckVersionMismatch,
+		Run: func(command *cobra.Command, args []string) {
+			execute.OnMultiCluster(restConfigProducer, execute.IfSubmarinerInstalled(show.All))
```

Comment:
Using `IfSubmarinerInstalled` here means we'll no longer show the broker details in a broker only cluster. To keep that behavior, we need to check if submariner is Installed after outputting the broker. 

```suggestion
			execute.OnMultiCluster(restConfigProducer, show.All)
```